cv32/sim/mcy/Makefile

```diff
@@ -0,0 +1,543 @@
+###############################################################################
```

Comment:
As far as I can tell, this Makefile makes no changes to the Makefile in ../core other than to add variables and targets for DIV_ONLY_FIRMWARE plus targets to run the mutated coded.   So it should be possible to reduce this makefile to `include $(PROJ_ROOT/cv32/sim/core/Makefile` and then add the aforementioned rules.